templates/cluster-template-aks.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: cluster.x-k8s.io/v1alpha3
+kind: Cluster
+metadata:
+  name: ${CLUSTER_NAME}
+  namespace: default
+spec:
+  clusterNetwork:
+    services:
+      cidrBlocks:
+      - 192.168.0.0/16
+  controlPlaneRef:
+    apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
+    kind: AzureManagedControlPlane
+    name: ${CLUSTER_NAME}-control-plane
+  infrastructureRef:
+    apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
```

Comment:
Is this in the `exp.infrastructure` group or is it in the `infrastructure` api group? In CAPI, `exp` is in a different api group.

This cause me a ton of pain in AzureMachinePool implementation, but currently have `exp.infrastructure.cluster.x-k8s.io/v1alpha3`.

Reply:
typo, I have a local copy with a bunch of uncommitted values for testing

Reply:
should be exp.infrastructure.cluster.x-k8s.io/v1alpha3 as in the Go I believe